lib/src/run_interactive_codemod.dart

```diff
@@ -289,6 +290,16 @@ int _runInteractiveCodemod(
           }
           if (choice == 'q') {
             logger.fine('applying patches');
+            var userSkipped = promptToHandleOverlappingPatches(appliedPatches);
+            // Store patch(es) to print info about skipped patches after codemodding.
+            skippedPatches.addAll(userSkipped);
+
+            // Don't apply the patches the user skipped.
+            for (var patch in userSkipped) {
```

Comment:
#nit could put this logic in a local function since it's duplicated here and below

Reply:
Gonna leave the nit -- I'd prefer not to modify the instance of `appliedPatches` and `skippedPatches` within a helper function as there’s already a lot of scope to keep in mind in `_runInteractiveCodemod`.